Content Security Policy: add a section related to CSP (#4976)
DevExpress/devextreme-documentation: concepts/Common/Security Considerations/40 Content Security Policy/00 Content Security Policy.md

> [Content Security Policy](https://developer.mozilla.org/ru/docs/Web/HTTP/CSP) (CSP) is a security feature that helps detect and mitigate certain types of attacks, such as clickjacking, cross-site scripting (XSS), and other malicious code injection attacks. The results of such attacks may be different: data theft, page spoofing, malware distribution, site defacement, and others. It is important to ensure that components of DevExtreme UI library are compatible with CSP.

Reviewer: ```suggestion
[Content Security Policy](https://developer.mozilla.org/ru/docs/Web/HTTP/CSP) (CSP) is a security feature that helps detect and mitigate certain types of attacks, such as clickjacking, cross-site scripting (XSS), and other malicious code injection attacks. The results of such attacks may be different: data theft, page spoofing, malware distribution, site defacement, and so on. It is important to ensure that components of the DevExtreme UI library are compatible with CSP.
```

> [Content Security Policy](https://developer.mozilla.org/ru/docs/Web/HTTP/CSP) (CSP) is a security feature that helps to detect and mitigate certain types of attacks, such as clickjacking, cross-site scripting (XSS), and other malicious code injection attacks. These attacks allow hackers to perform everything from data theft to page spoofing, malware distribution, or site defacement. It is important to ensure that components of DevExtreme UI library are compatible with CSP.

Reviewer 2: ```suggestion
[Content Security Policy](https://developer.mozilla.org/ru/docs/Web/HTTP/CSP) (CSP) is a security feature that helps detect and mitigate certain types of attacks, such as clickjacking, cross-site scripting (XSS), and other malicious code injection attacks. The results of such attacks may be different: data theft, page spoofing, malware distribution, site defacement, and others. It is important to ensure that components of DevExtreme UI library are compatible with CSP.
```

> If you want to apply CSP rules, you can define the `<meta>` tag and configure a policy in the following way:

Reviewer 2: ```suggestion
If you want to apply CSP rules, define a `<meta>` tag and configure a policy in the following way:
```

> [Content Security Policy](https://developer.mozilla.org/ru/docs/Web/HTTP/CSP) (CSP) is a security feature that helps to detect and mitigate certain types of attacks, such as clickjacking, cross-site scripting (XSS), and other malicious code injection attacks. These attacks allow hackers to perform everything from data theft to page spoofing, malware distribution, or site defacement. It is important to ensure that components of DevExtreme UI library are compatible with CSP.

Reviewer 3: Please wrap the link in <a> tag, not a markdown link, so it could be opened in a new tab (add "target: _blank")

Author: It opens in a new tab. I have checked its behavior.
